src/gui/scanfoldersdelegate.cpp

```diff
@@ -63,9 +63,9 @@ QWidget *ScanFoldersDelegate::createEditor(QWidget *parent, const QStyleOptionVi
     QComboBox* editor = new QComboBox(parent);
 
     editor->setFocusPolicy(Qt::StrongFocus);
-    editor->addItem(tr("Watch Folder"));
-    editor->addItem(tr("Default Folder"));
-    editor->addItem(tr("Browse..."));
+    editor->addItem(tr("Default save location"));
+    editor->addItem(tr("Same as monitored folder"));
+    editor->addItem(tr("Select folder..."));
```

Comment:
Hmmm, sorry, I didn't check thoroughly, this PR doesn't change it completely. There is a another place (ScanFoldersModel::data) that hold these strings. I'll make a PR later (and test it thoroughly).